app/models/search/bookmark_search_form.rb

```diff
@@ -141,8 +145,9 @@ def bookmarkable_search_results
 
   def sort_options
     [
-      ['Date Bookmarked', 'created_at'],
-      ['Date Updated', 'bookmarkable_date'],
+      ["Date Bookmarked", "created_at"],
+      ["Date Updated", "bookmarkable_date"],
+      ["Word Count", "word_count"]
```

Comment:
FYI these don't follow the convention for sentence case--do we want to change it? Or leave as-is?